schemas/notes.py

```diff
@@ -0,0 +1,29 @@
+from models.notes import NotesModel
+from models.user import UserModel
+from models.tickets import TicketModel
+from utils.time import time_format
+from marshmallow import fields, Schema, validates, ValidationError
+
+class NotesSchema(Schema):
+    id = fields.Integer()
+    text = fields.Str()
+    userid = fields.Integer()
+    ticketid = fields.Integer()
+
```

Comment:
Now that I see this. I really don't like it. haha

Reply:
I think we need to add the `require=True` for the `userid` and `ticketid` - I believe the auto-schema would've done that automagically.